### PR TITLE
Feature/page notfound

### PR DESCRIPTION
**Pull request by HegdeParikshith**
not found added
[Page-not-found-tracking](https://github.com/rumoursbehindme/react-client-app/issues/7)

### CODE REVIEW

**Comment by HegdeParikshith** [comment]
yee merge mado

**Review by rumoursbehindme — CHANGES_REQUESTED** [review]

File: src/screens/notfound/pagenotfound.tsx
[the remark this answers is not in the document]
Comment (HegdeParikshith) [reply src/screens/notfound/pagenotfound.tsx:1]: Page not found is not a component it is page/screen it is the best practice to keep it as page

File: src/screens/notfound/pagenotfound.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import notFound from '../../assets/Page not found.png'
+
+function PageNotFound() {
+  return (
+    <div className='screen-container' >
+        <h1>Page Not Found !</h1>
+      <img src={notFound} alt="logo" />
```
[the remark this answers is not in the document]
Comment (HegdeParikshith) [reply src/screens/notfound/pagenotfound.tsx:8]: as per react standards importing the assets is recommended. check the link below. [React best practice on importing assets](https://g.co/bard/share/135bdb128257)

File: src/assets/Page not found.png
[the remark this answers is not in the document]
Comment (HegdeParikshith) [reply src/assets/Page not found.png:1]: fixed

File: src/screens/notfound/pagenotfound.tsx
[the remark this answers is not in the document]
Comment (rumoursbehindme) [reply src/screens/notfound/pagenotfound.tsx:1]: > Page not found is not a component it is page/screen it is the best practice to keep it as page

- Make this as a component that can be reusable in many cases.  
- For example deprecation of route...
- Component should take the input message if not, it should show the default message.
- And please maintain these file naming standards  ex: 'page-not-found.tsx' and 'page-not-found.png'

File: src/screens/notfound/pagenotfound.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rumoursbehindme) [reply src/screens/notfound/pagenotfound.tsx:8]: Okay cool

File: src/assets/Page not found.png
[the remark this answers is not in the document]
Comment (rumoursbehindme) [reply src/assets/Page not found.png:1]: update the file name

File: src/shared/global-styles.css [hunk 2]
```diff
@@ -3,4 +3,16 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+        justify-content: center;
+        text-align: center;
+        /* Center text horizontally */
+    }
+    
+    /* Style for an image within the .screen-container */
+    .screen-container img {
```
Comment (rumoursbehindme) [inline src/shared/global-styles.css:15]: And also keep this  img CSS in the  same component folder

**Review by rumoursbehindme — CHANGES_REQUESTED** [review]

File: src/screens/notfound/pagenotfound.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rumoursbehindme) [reply src/screens/notfound/pagenotfound.tsx]: Since we cannot easily import the image from out side src folder ... I have added relative path to img src

**Review by rumoursbehindme — APPROVED** [review]

**Pull request merged**